tests/data/h5/createNeuron.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2013-2015, EPFL/Blue Brain Project
+#                          Juan Hernando <jheranando@fi.upm.es>
+#
+# This file is part of Brion <https://github.com/BlueBrain/Brion>
+
+#!/usr/bin/env python
+
+import h5py
+import numpy
+
+SOMA = 1
+AXON = 2
+BASAL = 3
+APICAL = 4
+
+out = h5py.File('test_neuron.h5', 'w')
+
+def createTBranchPoints():
+    # Creates a branch with a T shape
+    # The first section is a non-homogeneous sampling of 10 points on a straight
```

Comment:
Nope I am not against what you did. The use of phrase "non-homogeneous sampling of points" bothered me :) "Non-regular sampling" yes, "Non-homogeneous distance sampling" yes but sampling of point non-homogenus was a bit mis-leading.